apple/WCDB/abstract/expr.cpp

```diff
@@ -228,6 +228,15 @@ Expr Expr::concat(const Expr &operand) const
                               operand.m_description + ")");
     return expr;
 }
+    
+Expr Expr::substr(const Expr &start, const Expr &length) const
+{
+    Expr expr;
+    expr.m_description.append("SUBSTR(" + m_description + "," +
```

Comment:
You can use `Expr::Function` to package a SQL Core Function. See the `Expr::hex` for further info.